project/Dependencies.scala

```diff
@@ -4,19 +4,19 @@ object Dependencies {
   lazy val netlibVersion = "1.1.2"
   lazy val junitVersion = "4.13.1"
   lazy val thetaVersion = "1.1.3"
-  lazy val breezeVersion = "1.2"
+  lazy val breezeVersion = "2.0"
   lazy val scalaTestVersion = "3.2.2"
 
   // this is used in build.sbt so that it works with both 2.12 and 2.13
   lazy val scalaParallelCollectionsVersion = "1.0.0"
 
   val loloDeps = Seq(
-    "com.github.fommil.netlib" % "all" % netlibVersion pomOnly(),
-    "junit" % "junit" % junitVersion % "test",
-    "org.scalanlp" %% "breeze" % breezeVersion,
-    "io.citrine" %% "theta" % thetaVersion,
-    "org.scalatest" %% "scalatest" % scalaTestVersion % "test",
-    "com.novocode" % "junit-interface" % "0.11" % "test",
-    "org.knowm.xchart" % "xchart" % "3.5.2"
+    "com.github.fommil.netlib" % "all"             % netlibVersion pomOnly (),
```

Comment:
could this extra space after `pomOnly` be causing a problem?

Reply:
`scalafmt` is enforcing it being there, so I don't think thats the issue

Reply:
@latture Do you know if we need this dependency anymore?

Breeze >= 1.4 moved to a new version of netlib, not the `fommil` version. I am unsure what purpose this serves. But I do know that you dealt with this issue a few months ago.

Reply:
I have not. I should try that before we decide on merging this into Lolo.

Lolo was previously using breeze 1.2, so maybe this upgrade still will not conflict with `ecos` ?

Reply:
Hopefully there won't be any issues, but it's worth checking just to be safe.